components/automate-ui/src/app/entities/service-groups/service-groups.reducer.ts

```diff
@@ -11,7 +11,7 @@ import {
 
 export interface ServiceGroupEntityState {
   serviceGroups: ServiceGroup[];
-  serviceGroupHealthCounts: HealthSummary[];
+  serviceGroupHealthCounts: HealthSummary;
```

Comment:
We didn't need an array of `HealthSummary` but instead a single instance of that interface. Changed that here! 💯